include/ansi_code_markup.h

```diff
@@ -1,5 +1,7 @@
 /*
- *  Copyright (C) 2022  The DOSBox Staging Team
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2022-2022  The DOSBox Staging Team
```

Comment:
Probably should remain `2022`

Reply:
Result of me running this script several times :) Thanks for spotting this!

I added `git grep` invocation to the script to highlight such changes to the person updating copyright lines. Also, several files already had "2022-2022" before - I changed them to have a single year instead.